profiles/bicycle.lua

```diff
@@ -411,12 +409,10 @@ function way_function (way, result)
   if cycleway_right and profile.cycleway_tags[cycleway_right] then
     has_cycleway_right = true
   end
-  if cycleway and string.find(cycleway, "opposite") == 1 then
-    if reverse then
-      has_cycleway_right = true
-  else
-      has_cycleway_left = true
-    end
+  if cycleway and string.find(cycleway, "opposite") == 1 and not implied_oneway then
```

Comment:
i don't think it's correct to disregard contraflow on implied oneways. for example, a motorway implies oneway. cycleway=oppositve means that you can cycle in the backward direction, (since that's where the cycleway is), but not forward.